config/main.py

```diff
@@ -1583,7 +1618,14 @@ def delete_checkpoint(ctx, checkpoint_name, verbose):
 def list_checkpoints(ctx, verbose):
     """List the config checkpoints available."""
     try:
-        checkpoints_list = GenericUpdater().list_checkpoints(verbose)
+        if multi_asic.is_multi_asic():
```

Comment:
If the intention is to list the files, does original code still work?

Reply:
yes, you are correct, we dont have to do different operation for both delete and list action.